### PR TITLE
US-11-14

### PR DESCRIPTION
**Pull request by TaniaSakun**
This PR has the implementation for the Birthday module and contains the following User Stories:
1. US-11-create-method-to-add-birthday
2. US-12-create-method-to-edit-birthday
3. US-13-create-method-to-delete-birthday
4. US-14-create-method-to-get-birthday-by-duration

### CODE REVIEW

File: core/core/models/payload.py [hunk 1]
```diff
@@ -14,8 +14,12 @@ class ContactPayload(BaseModel):
 class BirthdayPayload(BaseModel):
     name: str
     birthday: Optional[str] = ''
-
-
+   
+    
+class AllBirthdaysPayload(BaseModel):
+    day_amount: Optional[str] = '7'
```
Comment (iMykhailychenko) [inline core/core/models/payload.py:20]: why string?

File: core/core/services/birthday.py [hunk 2]
```diff
@@ -1,9 +1,65 @@
...
+
+    return set_birthday(payload, record, birthday)
+
+
+@response()
+def get_birthdays_by_duration(payload):
+    records = database.all()
+    today = date.today()
+    records_with_this_week_birthday: list[Record] = []
+    days = int(payload.day_amount)
```
Comment (iMykhailychenko) [inline core/core/services/birthday.py:23]: lets make `payload.day_amount` as `int`, so we don't need to convert it here

File: core/core/models/payload.py (hunk 1 above)
Comment (TaniaSakun) [reply core/core/models/payload.py:20]: It wasn't a problem in the futurefor us to use an int? If no, sure I edit it

Comment (TaniaSakun) [reply core/core/models/payload.py]: Done

File: core/core/services/birthday.py (hunk 2 above)
Comment (TaniaSakun) [reply core/core/services/birthday.py]: Done

File: core/core/models/payload.py (hunk 1 above)
Comment (iMykhailychenko) [reply core/core/models/payload.py]: Yes, it should be okay. We can use any data type we want.

**Review by iMykhailychenko — APPROVED** [review]

**Pull request merged**